lib/matplotlib/quiver.py

```diff
@@ -87,21 +87,28 @@
 angles : {'uv', 'xy'} or array-like, default: 'uv'
     Method for determining the angle of the arrows.
 
-    - 'uv': Arrow direction in screen coordinates. Use this if the arrows
-      symbolize a quantity that is not based on *X*, *Y* data coordinates.
+    - 'uv': Arrow directions are based on screen coordinates (the coordinate
+       system of the plot, not the data). This is useful when the arrows represent 
+       a quantity that is not directly tied to the x and y data coordinates, such as
+       abstract physical quantities (e.g., velocities, forces).
```

Comment:
```suggestion
       a quantity whose direction is not tied to the x and y data coordinates.
```
I believe the examples are incorrect, e.g. velocities point into the direction of movement, therefore their direction is spatially related. Similar with forces. Actually, I believe most typical plots are spatially related, the only thing that comes to my mind right now that is not is a spin distribution, but that's too special for an example. If we don't have anything better, I suggest to leave the example out.